### PR TITLE
Library-wide: Use "meet" for non-concept type requirements

### PR DESCRIPTION
**Pull request by CaseyCarter**
...instead of "satisfy".

Partially addresses #1263.

### CODE REVIEW

**Comment by zygoloid** [comment]
@CaseyCarter Any chance you could rebase this in the next day or so, for the pre-Cologne working draft?

**Comment by CaseyCarter** [comment]
> @CaseyCarter Any chance you could rebase this in the next day or so, for the pre-Cologne working draft?

I'm taking a look now (I hadn't noticed the tag). It looks like mostly conflicts with the table label rework; this shouldn't be much trouble to fix up.

**Comment by CaseyCarter** [comment]
Rebased.

File: source/time.tex [hunk 1]
```diff
@@ -1335,7 +1335,7 @@
 The defaulted copy constructor of duration shall be a
 constexpr function if and only if the required initialization
 of the member \tcode{rep_} for copy and move, respectively, would
-satisfy the requirements for a constexpr function.
+meet the requirements for a constexpr function.
```
Comment (zygoloid) [inline source/time.tex]: This one should be "satisfy". See [dcl.constexpr]/3,4,6 and [class.copy.ctor]/12.

**Pull request merged**